config/locales/en.yml

```
@@ -9,6 +9,7 @@ en:
         ineligible_reason:
           misconduct: To teach in England, you must not have any findings of misconduct or restrictions on your practice.
           recognised: This is because you are not recognised as a school teacher in the country where you trained.
+          teach_children: You must have experience of teaching children who are aged somewhere between 5 and 16 years.
```

Is this content from the prototype? The `somewhere` seems incorrect:

```suggestion
          teach_children: You must have experience of teaching children who are aged between 5 and 16 years.
```